renderapi/transform.py

```diff
@@ -1370,6 +1370,127 @@ def estimate_dstpts(transformlist, src=None):
     return dstpts
 
 
+class NonLinearTransform(Transform):
+    """
+    render-python class that implements the mpicbg.trakem2.transform.nonLinearTransform class
+    
+    Parameters
+    ----------
+    dataString:str or None
+        data string of transformation
+    json:dict or NOne
+        json compatible dictionary representation of the transformation
+
+    Returns
+    -------
+    :class:`NonLinearTransform`
+        a transform instance
+
+
+    """
+
+    className = 'mpicbg.trakem2.transform.nonLinearTransform'

```

Comment:
It might make sense to add this to `renderapi.transform.load_leaf_json` as the default for loading lens correction transforms (unless we want to keep them generic and lighter weight)